src/main/java/fish/payara/extras/upgrade/UpgradeServerCommand.java

```diff
@@ -329,7 +331,7 @@ private void validateDistribution() throws CommandValidationException {
         // Check if distribution is defined.
         // Continue with upgrade if no defined distribution, user should be able rollback if needed.
         if (versionDistribution.isEmpty()) {
-            System.out.println("The distribution cannot be validated.");
+            logger.log(Level.INFO, "The distribution cannot be validated.");
```

Comment:
Is the INFO level appropriate? I understand the message as more important, System.out was always printed, so I would expect SEVERE...

Reply:
I'll change it to WARNING - SEVERE to me implies something has gone critically wrong, this message is just a "I don't know what you are so I'm going in blind" message